milton-processor/scribe.ts

```diff
@@ -35,7 +35,11 @@ app.get("/fetch", async (req, res) => {
   const format = req.query.format.toString();
   console.log(`Fetching cached raw contents of ${url} for ${format} format`);
   const cachedRawContents = await cache.fetchRawContents(url, format);
-  res.status(200).end(cachedRawContents, 'binary');
+  res.writeHead(200, {
+    'Content-Type': cachedRawContents.contentType,
+    'Content-Length': cachedRawContents.data.length,
```

Comment:
Out of curiosity, why do you set the content length?

Reply:
To indicate the size of the message body, in bytes, sent to the recipient. See the docs here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length :trollface:

Reply:
No but it sounds like express uses chunked encoding by default, and setting the Content-Length header disables that? Was there a reason why chunked encoding wasn't working?

Reply:
Oh interesting, didn't realize that... I think it's just because I was initially trying to figure out why the snapshots weren't loading, and thought maybe that was the reason, and then once I figured out that it was actually the content type being unset, didn't see a reason to remove it.

I dunno... I guess I could remove the content length for that benefit, but now that I've gone down this "chunked transfer encoding" rabbit hole, it seems like it doesn't apply for http2 anyways, which is what all of the clients I care about are using, and it *feels* nicer to set the content length since I already know it up front. Thoughts?

Reply:
I guess I have a slight preference for letting the framework do this for you (removes a potential bug, framework may be able to decide what to use depending on client / content), but it's really not that important. I was mostly curious